src/khoj/database/models/__init__.py

```diff
@@ -69,6 +72,35 @@ class Type(models.TextChoices):
     renewal_date = models.DateTimeField(null=True, default=None, blank=True)
 
 
+class ChatModelOptions(BaseModel):
+    class ModelType(models.TextChoices):
+        OPENAI = "openai"
+        OFFLINE = "offline"
+
+    max_prompt_size = models.IntegerField(default=None, null=True, blank=True)
+    tokenizer = models.CharField(max_length=200, default=None, null=True, blank=True)
+    chat_model = models.CharField(max_length=200, default="mistral-7b-instruct-v0.1.Q4_0.gguf")
+    model_type = models.CharField(max_length=200, choices=ModelType.choices, default=ModelType.OFFLINE)
+
+
+class Agent(BaseModel):
+    creator = models.ForeignKey(KhojUser, on_delete=models.CASCADE, default=None, null=True, blank=True)
```

Comment:
Should we add a comment saying creator is to be unset only for admin managed agents? Just as a reminder of why creator is allowed to remain unset